cosmos_service/README.md

```diff
@@ -63,3 +76,46 @@ $ curl "$URL/process/{job_id}/status"
 # Retrieve results
 $ curl -o output.zip "$URL/process/{job_id}/results"
 ```
+
+## Tests
+
+The `/test` directory contains a set of unit/integration tests for confirming COSMOS functionality against the running
+webservice. Certain test parameters, such as the base URL against which tests are run, are configured via
+[test/config.ini](https://github.com/UW-COSMOS/Cosmos/blob/master/cosmos_service/test/config.ini).
+
+### Running Tests
+
+Test prerequisites can be installed via `pip -r requirements.txt` from the `test` directory.
+Once prerequisites are installed, tests can be run with the `pytest` command:
+
+```bash
+$ cd test
+# install requirements
+$ pip install -r requirements.txt
+# run tests
+$ pytest
+```
+
+### Test cases
+
+Test cases for the COSMOS service are as follows:
+
+#### Annotations
+
+Annotations tests compare the bounding boxes and region labels generated by COSMOS against a 
+series of manually annotated PDF documents. Each test compares one of the two following metrics:
+
+* Count of labeled regions
+* Area covered by bounding boxes
+
+For the following 3 extraction labels:
+
+* Equations
+* Tables
+* Figures
+
+A separate test class is implemented per source document, with one metric/label pair per test. 
+Tests currently fail if the metric fails for the given label class on any page of the source document.
+Utility methods for writing tests against new source documents are included in `annotations/annotations_base.py`.
+The source bounding boxes for a document should be generated as XML by the manual annotation tool and placed in the
```

Comment:
We might want to include a link to the manual annotation tool git repository here if it's publicly available